README.adoc

```diff
@@ -132,6 +132,9 @@ Cheatsheets, reference material for tech stacks & computer science concepts.
  ** https://www.railstutorial.org/book[Ruby on Rails Tutorial by Michael Hartl]
  ** https://gist.github.com/jendiamond/5a26b531e8e47b4aa638[Rails Girls LA 2016]
 
+=== Career Advice
+ ** https://blog.alicegoldfuss.com/how-to-get-into-sre/[How to Get Into SRE]
+
```

Comment:
I'd place this right before [#command-line](https://github.com/LearnTeachCode/code-coffee-compendium/tree/ck-sre-career#command-line) as to keep the alphabetical ordering. Otherwise, lgtm!